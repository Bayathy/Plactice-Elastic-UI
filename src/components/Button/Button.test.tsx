import * as stories from "./button.stories";
import React from "react";
import { composeStories } from "@storybook/testing-react";
import { render, screen } from "@testing-library/react";

describe("Button", () => {
  const { Primary } = composeStories(stories);
  test("test toMatchSnapShot", () => {
    const { asFragment } = render(<Primary />);
    expect(asFragment()).toMatchSnapshot();
    screen.debug();
  });
});
