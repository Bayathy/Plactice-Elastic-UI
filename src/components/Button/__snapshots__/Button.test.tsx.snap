// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test toMatchSnapShot 1`] = `
<DocumentFragment>
  <button
    css="You have tried to stringify object returned from \`css\` function. It isn't supposed to be used directly (e.g. as value of the \`className\` prop), but rather handed to emotion so it can handle it (e.g. as value of \`css\` prop)."
  >
    test
  </button>
</DocumentFragment>
`;
